### ISSUE TITLE
z_est_bn = (z_est[l] - m) / v

### ISSUE THREAD

**Issue by daniadag** [rinuboney/ladder#22]
why we need to put this line in decoder in line 179???  after you apply batch normalization 

it seems like you apply mean and v for output layer before and after g_gauss
u = batch_normalization(u)
z_est[l] = g_gauss(z_c, u, layer_sizes[l])
z_est_bn = (z_est[l] - m) / v
